mastracode/src/web/ui/styles.css

```
@@ -2760,6 +2760,7 @@ body::before {
 .provider-tick {
   flex-shrink: 0;
 }
+.provider-tick.oauth,
 .provider-tick.stored {
   color: var(--green);
 }
```

Stylelint (17.13.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)